lib/assets/javascripts/cartodb/common/dialogs/delete_column/delete_column_view.js

```diff
@@ -57,7 +56,7 @@ module.exports = BaseDialog.extend({
 
   ok: function() {
     this._panes.active('loading');
-    this.table.deleteColumn(this.column);
+    this.table.deleteColumn(this.options.column);
```

Comment:
How come you need this change? You haven't added anything to the table model's `options` right?